protocol-units/settlement/mcr/contracts/src/staking/MovementStaking.sol

```diff
@@ -487,7 +454,7 @@ contract MovementStaking is
         address[] calldata attesters,
         uint256[] calldata amounts,
         uint256[] calldata refundAmounts
-    ) public {
+    ) public nonReentrant {
```

Comment:
Is slashing safe? not sure if I'm following the logic that would prevent arbitrage slashing